### DISCUSSION TITLE
Set default timeframe for dashboard

### DISCUSSION THREAD

**Discussion by heybobjones** [umami-software/umami#2804] [Ideas]
The dashboard seems to default to a 24 hours view, it would be great if we can set it to whatever we want a s a default.

**Comment by rsynnot** [comment]
Would really like this option to set a detault in the account settings area.

**Comment by MarcoGriep88** [comment]
Me too

**Comment by d-flood** [comment]
My team would like this feature as well. Would there be openness for a PR to set it as a config variable if not in account settings?

**Reply by ghost** [reply]
Made an PR for solving this https://github.com/umami-software/umami/pull/3746

**Comment by ghost** [comment]
Made an PR for solving this https://github.com/umami-software/umami/pull/3746
